src/script/components/VerificationBadge/VerificationBadges.tsx

```diff
@@ -243,6 +261,8 @@ export const VerificationBadges = ({
   if (!MLSStatus && !isProteusVerified) {
     return null;
   }
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  const id = useRef(new Date().getTime());
```

Comment:
I believe it's about calling hooks conditionally? Can't we just move it above the first return statement and live in peace with react and eslint? :D